ccdaservice/serveccda.js

```diff
@@ -11,7 +11,7 @@
 
 "use strict";
 
-const enableDebug = false;
+const enableDebug = true;
```

Comment:
hi Jerry, looks like we're running in debug now :)

Reply:
Did so on purpose. Not intrusive and no diff than logging. It gives a copy of the xml and the json the doc was built using in sites.../temp/cad.xml cda.json and will overwrite itself. I do this on major updates for a while so I can troubleshoot to help folks.

Reply:
think the problem with that is that it potentially leaves phi outside of the documents folder if I remember well

Reply:
No different than any other report being saved in site documents. i.e \openemr\sites\default\documents\temp